### ISSUE TITLE
Error in review from the patient portal

### ISSUE THREAD

**Issue by luisuriarte** [openemr/openemr#7098]
### Describe the bug

When doing the review by the patient in the patient portal. It gives an error when checking in the openemr user.

Error in review from the patient portal When doing the review by the patient in the patient portal. It gives an error when checking in the openemr user.
![image](https://github.com/openemr/openemr/assets/10144928/465601d4-7ecb-4a74-b325-6dd5007dba1b)

![image](https://github.com/openemr/openemr/assets/10144928/3482fd80-36c1-463c-a098-bdf557614c14)

This is reproduced in the official demos

**Comment by sjpadgett** [comment]
@luisuriarte What PHP version? Which demo? Did you check php error logs?

**Comment by luisuriarte** [comment]
In https://demo.openemr.io/

**Comment by sjpadgett** [comment]
@luisuriarte I'm unable to recreate on demo or my windows dev using PHP 8.2.
I have however uncovered a pathing issue on windows that doesn't appear on linux on login.
If you can recreate again let me know and try to get me a php error by turning on User Debug in Config.

**Comment by sjpadgett** [comment]
@luisuriarte 
This error is caused because you were using the portal and then logged into OpenEMR from the same user agent.
Because you have an active portal session running when log into OpenEMR by which action will open its session and then trying to access audits(review) from the Patient Dashboard Portal quick Audit link will result in the above error. 
Because that work flow calls audits via an API request and since we have an active portal session still running the request will actively fail for security reasons.
However if the request comes from the Portal Dashboard Audits action, I will trap the error and destroy the portal session. I did this as a courtesy for folks testing portal audits e.t.c.. I won't do that for the API call because that call could originate outside normal authorized sessions. Again for security. Simply, active portal and openemr sessions can not be shared. Portal session auth is not compatible.

Best to log out of portal before using OpenEMR or use two different user agents(browsers e.g. chrome and FF) ie. sessions are isolated from each other.

Hope this answers you for this issue. It didn't come to naught though as I found that some extraneous characters are leaking into the HTML when using the Portal Dashboard so yeah!:)
